test/cpp/end2end/xds/xds_cluster_end2end_test.cc

```diff
@@ -349,23 +353,7 @@ TEST_P(CdsTest, ClusterChangeAfterAdsCallFails) {
   cluster.mutable_eds_cluster_config()->set_service_name(kNewEdsResourceName);
   balancer_->ads_service()->SetCdsResource(cluster);
   // Make sure client sees the change.
-  // TODO(roth): This should not be allowing errors.  The errors are
```

Comment:
Note that this TODO should have actually been addressed as part of #28128, but I didn't remember to do that at the time.